lib/nexaas_id/resources/widget.rb

```diff
@@ -25,4 +24,16 @@ class NexaasID::Resources::Widget < NexaasID::Resources::Base
   def navbar_url
     %(#{configuration.url}/api/v1/widgets/navbar?access_token=#{api.token})
   end
+
+  # Retrieves the user's widget URL
+  #
+  # [API]
+  #   Method: <tt>GET /api/v1/widgets/user</tt>
+  #
+  #   Documentation:
+  #
+  # @return [String] user's widget URL
+  def widget_url
+    %(#{configuration.url}/api/v1/widgets/user.js?access_token=#{api.token}&callback=initWidget)
```

Comment:
Só uma dúvida: nós vamos forçar o *callback* com nome de `initWidget` mesmo? Na [documentação](https://docs.id.nexaas.com/api/widget), o nome é apenas um exemplo: o nome da função JS logo acima.

Sugiro que o *callback* seja parametrizado (pode até ter `initWidget` como *default*).

Reply:
Estava considerando que esse valor fosse padrão. Acho a idéia do _default_ bacana.